img_styler/image_prompt/dalle_mini_model.py

```diff
@@ -0,0 +1,98 @@
+# Reference: https://colab.research.google.com/github/borisdayma/dalle-mini/blob/main/tools/inference/inference_pipeline.ipynb#scrollTo=sOtoOmYsSYPz
+
+import os
+import toml
+from pathlib import Path
+from loguru import logger
+
+import numpy as np
+import jax
+import jax.numpy as jnp
+from PIL import Image
+from dalle_mini import DalleBart, DalleBartProcessor
+from vqgan_jax.modeling_flax_vqgan import VQModel
+
+# Load the config file to read in system settings.
+base_path = (Path(__file__).parent / "../configs/").resolve()
+app_settings = toml.load(f"{base_path}/settings.toml")
+
+
+class DalleMini:
```

Comment:
Was able to get it working. Tried re-producing few concepts from --> https://openai.com/blog/dall-e/
<img width="506" alt="Screen Shot 2022-12-14 at 9 41 06 AM" src="https://user-images.githubusercontent.com/1318029/207668201-4c48de34-0cbe-49e6-b3a5-38965131a875.png">

**Observation/Questions:**
- Call to `p_generate` seemed slow. It could be the hardware I m using, though. Wondering if there are tricks to make it faster?
- If we swap DALLE-mini weights with `-mega` n `settings.toml` update then things should work, right? 
- Also, throws an exception if seed is `empty` default `None`

Reply:
* It was quite slow on my end as well, but it was faster than the case with stable diffusion. Any chance it is not using the GPU? I think it would give a warning if that's the case, but it doesn't error out. There were several steps to install JAX for GPU, which I hadn't mentioned in the description.
* Yes, I think it should. The mega model was too large for my memory so I wasn't able to properly test it out.
* Ah, will fix it before merging.